tests/utils/test_deprecated_kwargs.py

```diff
@@ -42,6 +42,45 @@ def simple_sum(alpha, beta, a=0, b=0):
         simple_sum(1, 2, **func_kwargs)
 
 
+@pytest.mark.utils
+@pytest.mark.parametrize(
+    "arguments, message, func_kwargs, msg_expected",
+    [
+        (
+            ["a"],
+            "The keyword argument '{argument}' of '{func_name}' is deprecated",
+            dict(a=1),
+            "The keyword argument 'a' of 'simple_sum' is deprecated",
+        ),
+        (
+            ["b"],
+            "The keyword argument '{argument}' of '{func_name}' is deprecated",
+            dict(b=2),
+            "The keyword argument 'b' of 'simple_sum' is deprecated",
+        ),
+        (
+            ["a", "b"],
+            "The option '{argument}' of '{func_name}' is deprecated.",
+            dict(a=1, b=2),
+            "The option 'a' of 'simple_sum' is deprecated.",
+        ),
+        (
+            ["b", "a"],
+            "The keyword of function is deprecated.",
+            dict(a=1, b=2),
+            "The keyword of function is deprecated.",
+        ),
```

Comment:
@Zeroto521 can I check, is this the appropriate behaviour? I would have thought that the expected behaviour would be something like "The keyword arguments 'a' and 'b' of function 'simple_sum' is deprecated."